scripts/validation/validate_labels.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+
+import sys
+
+from oremda.clients import client_from_image
+
+
+if len(sys.argv) < 2:
+    sys.exit('Usage: <script> <image>')
+
+image_path = sys.argv[1]
+client = client_from_image(image_path)
+
+image = client.image(image_path)
+image.validate_labels()
+print('Labels are valid!')
```

Comment:
This script validates labels and can be used like:

```bash
./validate_labels.py oremda/test:latest
```

for docker images, or:

```bash
./validate_labels.py path/to/oremda_test.simg
```

for singularity images.